src/domain/community/organization/organization.module.ts

```diff
@@ -27,12 +29,13 @@ import { PlatformAuthorizationPolicyModule } from '@src/platform/authorization/p
     OrganizationVerificationModule,
     UserModule,
     UserGroupModule,
-    TagsetModule,
+    EntityResolverModule,
```

Comment:
Not sure this module is needed, it likely can be removed. It may be left over from when I was doing the lookups directly on entities.